package.json

```diff
@@ -75,7 +75,7 @@
     "min-wd": "^2.11.0",
     "mocaccino": "^4.1.0",
     "mocha": "^5.2.0",
-    "puppeteer": "^1.13.0",
+    "puppeteer": "^1.19.0",
```

Comment:
Puppeteer changelog looks unsuspicious :+1:

Reply:
Thanks for looking through. I didn’t wait with merging because master was failing again and I couldn’t get it green and this change worked in every environment all the time ¯\_(ツ)_/¯